app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/presenter/AuthenticationPresenter.kt

```diff
@@ -1,6 +1,9 @@
 package com.github.HumanLearning2021.HumanLearningApp.presenter
 
+import android.content.Context
+import android.content.Context.MODE_PRIVATE
 import android.content.Intent
+import android.content.SharedPreferences
```

Comment:
you can optimize import I think?

Reply:
If you're using the saveAction plugin it should do it automatically I think.